pinot-servicemanager/schemas/spanEventView-schemaFile.json

```diff
@@ -131,6 +131,18 @@
       "name": "api_exit_calls",
       "dataType": "INT",
       "defaultNullValue": 0
+    },
+    {
+      "name": "api_callee_name_count__KEYS",
```

Comment:
Note: we have added an inverted index for the map column - Tags. Not sure, it will be needed here of __KEY column. we can add it later as well if needed.